modules/gitops-argo-cd-properties.adoc

```diff
@@ -9,136 +9,118 @@
+  * _<redisProxyVersion>_ - The tag to use for the Redis HAProxy container image.
+
+|`helpChatURL`    |URL for getting chat help (this will typically be your Slack channel for support).|`https://mycorp.slack.com/argo-cd` |
+|`helpChatText`    |The text that appears in a text box for getting chat help.|`Chat now!`|
+|`image`    |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.|`argoproj/argocd` |
+|`ingress`    |Ingress configuration options.| `__<object>__` |
+|`initialRepositories`    |Initial Git repositories to configure Argo CD to use upon creation of the cluster.|`__<empty>__` |
+
+|`notifications`    |Notifications controller configuration options.|`__<object>__`
+a|* _<enabled>_ - The toggle to start the notifications-controller.
+  * _<image>_ - The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.
+  * _<version>_ - The tag to use with the Notifications container image.
+  * _<resources>_ - The container compute resources.
+  * _<logLevel>_ - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+
+|`repositoryCredentials`    |Git repository credential templates to configure Argo CD to use upon creation of the cluster.| `__<empty>__` |
+
+|`initialSSHKnownHosts`    |Initial SSH Known Hosts for Argo CD to use upon creation of the cluster.| `__<default_Argo_CD_Known_Hosts>__` |
+
+|`kustomizeBuildOptions`    |The build options and parameters to use with `kustomize build`.|`__<empty>__` |
+
+|`oidcConfig` |The OIDC configuration as an alternative to Dex.|`__<empty>__` |
+
+|`nodePlacement` |Add the `nodeSelector` and the `tolerations`.|`__<empty>__` |
+
+|`prometheus` |Prometheus configuration options.|`__<object>__`
+a|* _<enabled>_ - Toggle Prometheus support globally for Argo CD.
+  * _<host>_ - The hostname to use for Ingress or Route resources.
+  * _<ingress>_ - Toggles Ingress for Prometheus.
+  * _<route>_ - Route configuration options.
+  * _<size>_ - The replica count for the Prometheus `statefulSet`.
+
+|`rbac` |RBAC configuration options.|`__<object>__`
+a|* _<defaultPolicy>_ - The `policy.default` property in the `argocd-rbac-cm` config map. The name of the default role which Argo CD will fall back to, when authorizing API requests.
+  * _<policy>_ - The `policy.csv` property in the `argocd-rbac-cm` config map. CSV data containing user-defined RBAC policies and role definitions.
+  * _<scopes>_ - The scopes property in the `argocd-rbac-cm` config map. Controls which OIDC scopes to examine during RBAC enforcement (in addition to sub scope).
+
+|`redis` |Redis configuration options.|`__<object>__`
+a|* _<autoTLS>_ - Use the provider to create the Redis server's TLS certificate (one of: OpenShift). Currently only available for {product-title}.
```

Comment:
Because of the repo changes the {product-title} now stands for GitOps instead of OCP. The issue is visible here.
I would also reformulate this one a little bit so that it is clear that the only currently available option is `openshift`.
So for example:
```suggestion
a|* _<autoTLS>_ - Use the provider to create the Redis server's TLS certificate. Only the `openshift` value is currently available.
```

Reply:
Because of the {product-title} vs {OCP} issue, I would recommend creating a Jira that would go through the whole GitOps documentation to find all the cases where {OCP} should be used instead of {product-title}.
cc: @Srivaralakshmi

Reply:
Thanks @eromanova97 for the observation and request - Here is the JIRA: https://issues.redhat.com/browse/GITOPS-4168

Reply:
@joaedwar you can fix this instance mentioned by @eromanova97 in the GitOps assemblies and modules you are working.